src/game/save_file.c

```diff
@@ -52,6 +52,12 @@ s8 gLevelToCourseNumTable[] = {
 STATIC_ASSERT(ARRAY_COUNT(gLevelToCourseNumTable) == LEVEL_COUNT - 1,
               "change this array if you are adding levels");
 #ifdef EEP
+#include "vc_check.h"
+#include "vc_ultra.h"
+
+// Note that for safety reasons use IS_VC() instead gIsVC when working with EEP
```

Comment:
Can you move the assignment for gIsVC earlier rather than calling IS_VC every time?

Reply:
To be fair IS_VC() is a really cheap check and considering it is done the saving there are no real performance concerns - it is probably about as slow as reading from RAM. I will check what I can do though